web/router.ex

```diff
@@ -98,6 +98,7 @@ defmodule Ask.Router do
     get "/registrations/confirmation_sent", Coherence.RegistrationController, :confirmation_sent
     get "/registrations/confirmation_expired", Coherence.RegistrationController, :confirmation_expired
     get "/passwords/password_recovery_sent", Coherence.PasswordController, :password_recovery_sent
+    get "/mobile_survey", MobileSurveyController, :index
```

Comment:
We should move `/mobile_survey` to a scope of its own, since the `Coherence.Auth.Session` plug in the `browser` pipeline does not apply here.